test/fixtures/env-app/boot/foo.env.js

```diff
@@ -0,0 +1 @@
+process.bootFlags.push('fooLoadedInEnv');
```

Comment:
AFAICT, this file is not used by your new test, please remove. I believe the tests is using `boot/test/bar.js` instead, isn't?